### ISSUE TITLE
Userscript Sync does not work

### ISSUE THREAD

**Issue by langningchen** [Tampermonkey/tampermonkey#2188]
When using `Userscript Sync` to export scripts, 

## Expected Behavior

Export the `.meta.json` and `.user.js` file to the specified cloud drive.

## Actual Behavior

The content of the files are `[object Object]` instead of something meaningful.

![image](https://github.com/user-attachments/assets/c19bf5c5-7366-49f4-9004-e41a0309ad1e)

![image](https://github.com/user-attachments/assets/f60668a1-e767-4377-a6d6-8760f0da5fc8)

![image](https://github.com/user-attachments/assets/b4cd224d-d517-4c2b-b0a3-3fb2ebe2ae82)

----------

And when we try to import it, we get these logs: 
```
sync: schedule sync for run in 1 ms
background.js:687 si: unable to parse extended info of undefined
background.js:731 sync: finished
```

## Specifications

- Chromium: 129.0.2792.65
- TM: 5.3.6212
- OS: Windows Version 22H2 OS Build 19045.4957

**Comment by langningchen** [comment]
And I've found that the latest stable version (v5.1.1) does not have this issue. :joy:

**Comment by derjanb** [comment]
Should be fixed at 5.3.6213 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|xpi in review)

Please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions` (after you've enabled 'Developer Mode').

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by langningchen** [comment]
It works! Thanks.
